pages/common/rustc.md

```diff
@@ -0,0 +1,19 @@
+# rustc
+
+> The Rust compiler.
+
+- Compile a single file:
+
+`rustc {{file.rs}}`
+
+- Compile with high optimization:
+
+`rustc -O {{file.rs}}`
+
+- Compile with debugging information:
+
+`rustc -g {{file.rs}}`
+
+- Build a test harness:
```

Comment:
Any chance we could use a more beginner-friendly description? Maybe a bit more descriptive would help.